consensus/hotstuff/verification/combined_signer_v2_test.go

```diff
@@ -53,18 +58,33 @@ func TestCombinedSignWithBeaconKey(t *testing.T) {
 
 	committee := &mocks.DynamicCommittee{}
 	committee.On("DKG", mock.Anything).Return(dkg, nil)
+	committee.On("Self").Return(me.NodeID())
+	committee.On("IdentityByBlock", fblock.Header.ID(), fblock.Header.ProposerID).Return(proposerIdentity, nil)
+	committee.On("LeaderForView", fblock.Header.View).Return(signerID, nil).Maybe()
```

Comment:
not sure; to me the following would be a bit more intuitive 

```suggestion
	committee.On("LeaderForView", proposerView).Return(signerID, nil).Maybe()
```